bundles/binding/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/things/VeluxProductState.java

```diff
@@ -1,5 +1,4 @@
-/**
- * Copyright (c) 2010-2018 by the respective copyright holders.
+/** * Copyright (c) 2010-2018 by the respective copyright holders.
```

Comment:
Not sure what happened to this copyright header, but please revise it to match the other ones.